build_no_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+mkdir -p src/main/resources/static
+rm -r src/main/resources/static/* || true
+rm -r ui/build/* || true
+cd ui
+npm install
+npm run build
+cd ..
+cp -r ui/build/* src/main/resources/static
+rm -r ui/build/* || true
+
+mvn clean install -Dmaven.test.skip=true
```

Comment:
This still runs the docker compose up and downs, it needs `-DdockerCompose.skip` too